src/dev/flang/be/c/C.java

```diff
@@ -605,6 +605,26 @@ private void createCode(CFile cf)
                                                  r.ret()))));
     var ordered = _types.inOrder();
 
+    // start of fileio related intrinsics
+    // fuzion.std.fileio.read
+    // currently logs console messages instead of proper returns in case of errors
+    cf.print
+      (CStmnt.lineComment("helper to read bytes from a file"));
+      cf.print
+        ("void "+CNames.FILEIO_READ._name+"(char* path, void* content, size_t size)\n"+
+          "{FILE *f = fopen(path, \"r\");\n"+
+          "if (f == NULL)\n"+
+          "{printf(\"error happened\");\n"+
```

Comment:
please add a proper error message. Where did it happen? what happened? Also, please print error to stderr. Start the printed message with `*** `.